src/main/kotlin/net/atos/zac/task/TaskService.kt

```diff
@@ -151,14 +151,22 @@ class TaskService @Inject constructor(
         }
         val taskIds = mutableListOf<String>()
         restTaakVrijgevenGegevens.taken.forEach {
-            flowableTaskService.readOpenTask(it.taakId).let { task ->
-                releaseTask(
-                    task = task,
-                    loggedInUser = loggedInUser,
-                    reden = restTaakVrijgevenGegevens.reden
+            try {
+                flowableTaskService.readOpenTask(it.taakId).let { task ->
+                    releaseTask(
+                        task = task,
+                        loggedInUser = loggedInUser,
+                        reden = restTaakVrijgevenGegevens.reden
+                    )
+                    sendScreenEventsOnTaskChange(task, it.zaakUuid)
+                    taskIds.add(task.id)
+                }
+            } catch (taskNotFoundException: TaskNotFoundException) {
+                LOG.log(
+                    Level.SEVERE,
```

Comment:
Is this really SEVERE? I see this as WARNING as we are able to work around it and it does not impact functionality. No open task might also mean that meanwhile it was closed I think

Reply:
>it does not impact functionality.

I think it does in most cases. If there is a closed task in the werkvoorraad that is really a serious issue.